lib/ssl/doc/src/using_ssl.xml

```diff
@@ -41,46 +41,52 @@
   'tlsv1.3')</c> </seemfa>.  The available cipher suites for a
   connection depend on the TLS version and pre TLS-1.3 also on the
   certificate. To see the default cipher suite list change <c>all</c>
-  to <c>default</c>.  Note that TLS 1.3 and previous versions does not
+  to <c>default</c>.  Note that TLS 1.3 and previous versions do not
   have any cipher suites in common, for listing cipher suites for a
   specific version use <seemfa
   marker="ssl:ssl#cipher_suites/2"><c>ssl:cipher_suites(exclusive,
   'tlsv1.3')</c> </seemfa>.  Specific cipher suites that you want your
   connection to use can also be specified. Default is to use the
   strongest available.</p>
-  
+
+
+  <p>The following sections shows small examples of how to set up
+  client/server connections using the Erlang shell. The returned
+  value of the <c>sslsocket</c> is abbreviated with <c>[...]</c> as
+  it can be fairly large and is opaque to the user except for the
+  purpose of pattern matching.</p>
+
+
+  <note><p>Note that client certificate verification is optional for the server and needs additional conguration
+    on both sides to work. Also note that the option <c>certs_keys</c> is introduced in OTP-25.
```

Comment:
_Also note that the option <c>certs_keys</c> is introduced in OTP-25._

Or maybe simplified:
_See also <c>certs_keys</c> option introduced in OTP-25._ ?

Reply:
Reformulated!